ci.jsonnet

```diff
@@ -520,6 +520,9 @@
     {name: "ruby-test-compiler-graal-core"} + linux_gate + $.graal_core + {run: jt(["test", "compiler"])},
     # {name: "ruby-test-compiler-graal-enterprise"} + linux_gate + $.graal_enterprise + {run: jt(["test", "compiler"])},
     # {name: "ruby-test-compiler-graal-vm-snapshot"} + linux_gate + $.graal_vm_snapshot + {run: jt(["test", "compiler"])},
+
+    {name: "ruby-test-compiler-graal-core-java9"} + linux_gate + $.graal_core + labsjdk9 + {run: jt(["test", "compiler"])},
+    {name: "ruby-test-compiler-standalone-java9"} + linux_gate + $.graal_core + labsjdk9 + {run: ["bin/truffleruby", "-J-XX:+UnlockExperimentalVMOptions", "-J-XX:+EnableJVMCI", "-J--module-path=../graal/sdk/mxbuild/modules/org.graalvm.graal_sdk.jar:../graal/truffle/mxbuild/modules/com.oracle.truffle.truffle_api.jar:../graal/compiler/mxbuild/modules/jdk.internal.vm.compiler.jar", "-J-Dgraal.TraceTruffleCompilation=true", "-e", "raise 'no graal' unless Truffle.graal?"]},
```

Comment:
Could you extract this in a separate job definition in a local var and split lines?

Reply:
I didn't think it was interesting enough to spell out like that. Do you think I should?

Reply:
Yes, because it's very hard to review/read in this form.
But also, I think it's good to keep the `builds:` list tidy and avoid long lines with command lines in it, unless it's really one short command.